src/DataTable/selection/ControlledSelectionStatus.jsx

```diff
@@ -49,11 +48,13 @@ const ControlledSelectionStatus = ({ className, clearSelectionText }) => {
 
 ControlledSelectionStatus.defaultProps = {
   className: undefined,
-  clearSelectionText: CLEAR_SELECTION_TEXT,
+  clearSelectionText: undefined,
 };
 
 ControlledSelectionStatus.propTypes = {
+  /** A class name to append to the base element */
   className: PropTypes.string,
+  /** A text that appears on the `Clear selection` button, defaults to 'Clear Selection' */
   clearSelectionText: PropTypes.string,
```

Comment:
Should this also handle elements in addition to strings as well?

Reply:
it should, thanks, updated